apps/evm/lib/evm/operation/environmental_information.ex

```diff
@@ -206,7 +206,7 @@ defmodule EVM.Operation.EnvironmentalInformation do
         machine_state: machine_state
       }) do
     if size == 0 do
-      0
+      %{machine_state: machine_state}
```

Comment:
Not sure if it's worth it or not. What do you think about adding a test to `environmental_information_test.exs`?

Reply:
Yep, good call: [
Add a test for EnvironmentalInformation.codecopy where size == 0](https://github.com/poanetwork/mana/pull/372/commits/079c24cd71602f892977af208410fc8dcca0f595)